spec/javascript/e2e/ApplicationNewPage.e2e.js

```diff
@@ -34,7 +34,7 @@ describe('ApplicationNewPage', () => {
     expect(values).toContain(LAST_NAME)
     expect(values).toContain(DATE_OF_BIRTH)
 
-    browser.close()
+    await browser.close()
```

Comment:
All of these `await`s that you see added in this PR before the `browser.close()`s in our e2e tests are to ensure that we actually wait for the browser to close! Not having these awaits is what was causing the browsers in the e2e tests to never actually close and causing the whole test process to hang for a long time after finishing the tests.